clang/include/clang/Config/config.h.cmake

```diff
@@ -99,6 +99,7 @@
 #define KITSUNE_REALM_LIBRARY_DIR        "${REALM_LIBRARY_DIR}"
 #define KITSUNE_REALM_LINK_LIBS          "${REALM_LINK_LIBS}"
 
+/* TODO: maybe use LIBOMP_XXX here? */
```

Comment:
This may be spurious.  Can't remember what I was thinking about when I wrote it...

Reply:
Fixed